From 9cee8e2a53fbb57cd343e53105f6398788b03818 Mon Sep 17 00:00:00 2001
From: Tom Englund <tomenglund26@gmail.com>
Date: Mon, 14 Dec 2020 16:07:40 +0100
Subject: [PATCH] turn on fhd display with VCC3P

---
 src/northbridge/intel/sandybridge/gma.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/northbridge/intel/sandybridge/gma.c b/src/northbridge/intel/sandybridge/gma.c
index 86e12b0c64..e0c7caef1e 100644
--- a/src/northbridge/intel/sandybridge/gma.c
+++ b/src/northbridge/intel/sandybridge/gma.c
@@ -526,6 +526,10 @@ static void gma_pm_init_post_vbios(struct device *dev)
 		gtt_write(0xc4030, reg32);
 	}
 
+	/* Turn the panel power on using eDP VDD Override */
+	gtt_write(PCH_PP_CONTROL, gtt_read(PCH_PP_CONTROL) | PCH_PP_UNLOCK | EDP_FORCE_VDD);
+	udelay(125000); /* And wait until it powers up */
+
 	/* Setup Panel Power On Delays */
 	reg32 = gtt_read(0xc7208);
 	if (!reg32) {
-- 
2.29.2

